### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by EreckGordon**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Waneiac</title>
```
Comment (lazarljubenovic) [inline src/index.html]: I'm sorry but cats are not eligible to signing the CLA. Therefore, I cannot accept this PR.

**Review by lazarljubenovic — CHANGES_REQUESTED** [review]
:cat2:

File: src/index.html (hunk 1 above)
Comment (frederikprijck) [reply src/index.html]: I can take over this PR to get it accepted if you want as I aint no cat.

Comment (lazarljubenovic) [reply src/index.html]: I don't usually allow it but this time I'll _leggo_.

**Review by frederikprijck — APPROVED** [review]
LGTM

**Comment by EreckGordon** [comment]
>Lazar Ljubenović @lazarljubenovic Jul 11 14:02
wow the title says Title
someone do a pr for that i cant be arsed
https://github.com/wane/site

PR fully complies with request.

**Comment by lazarljubenovic** [comment]
As a whiney client, I have a change request. Please use a different title, eg. one which is acceptable to hoomans.

**Comment by EreckGordon** [comment]
done

**Comment by lazarljubenovic** [comment]
Thanks! Sent for internal review and testing, will report back in a few years.

**Comment by EreckGordon** [comment]
zero fun, sir!

**Comment by lazarljubenovic** [comment]
This is terrible. Who names their project _Wane_?!

**Review by lazarljubenovic — APPROVED** [review]

**Pull request merged**